src/pages/Admin/Charity/EditProfile/Form.tsx

```diff
@@ -42,7 +42,7 @@ export default function Form() {
       className="w-full max-w-4xl justify-self-center grid content-start gap-6 mt-6 font-body"
     >
       <Link
-        to={`${appRoutes.marketplace}/${id}`}
+        to={`${appRoutes.profile}/${id}`}
```

Comment:
Wrong URL

Reply:
`marketplace/${id}` is the correct URL for accessing a single profile page.

Reply:
@SovereignAndrey it was `/profile/{id}` before I transitioned to smart contract side, not sure if something changed in the meantime @ap-justin ?

Reply:
> @SovereignAndrey @ap-justin it was `/profile/{id}` before I transitioned to smart contract side, not sure if something changed in the meantime?

Yes, it changed. If you click a profile card in the marketplace, you'll see that everything profile-related is now nested under the `/marketplace/{id}`. The old endpoint `/profiles/{id}` still exists, serving as a redirect for old Juno profile links out in the wild, so that they map to their linked NEW endowment ID profile.

Reply:
@SovereignAndrey must have been some unrelated issue that was causing errors on my side (which prompted me to make this change).

Reverted (+ removed a duplicate Marketplace route declaration), see https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2480/commits/d96078d4275729e0bd5670e5b692a1dd7db716ce